### ISSUE TITLE
Alpha Loss and Weight Loss

### ISSUE THREAD

**Issue by stillbetter** [JuewenPeng/MPIB#7]
Hi Juewen, I want to know  the specific implementation of these two loss functions? L1 regularization means   `l1_norm = torch.norm(alpha, p=1)` or `torch.nn.L1Loss(alpha,1)`?
